feedback-module/src/components/Module.js

```diff
@@ -67,20 +71,17 @@ function Module({ pagetitle, endpoint, dir }) {
     }
   }, [screen]);
 
-  // sendFormData determines which form data state to update, based on the formID
-  const sendFormData = (formID) => {
-    if (formID === "feedback") {
-      updateFeedbackForAPI(processFeedback, [checkedFields, inputQuestions]);
-      requestService("feedback", {
-        id: endpoint,
-        feedback: feedbackForAPI,
-      });
-      console.log(feedbackForAPI);
-    } else if (formID === "research") {
-      trackFutureResearch();
-      updateUserInfo(processUserInfo, [inputQuestions, endpoint]);
-      requestService("userResearch", userInfo);
-      console.log(userInfo);
+  // requestInfo returns the object to be submitted
+  const requestInfo = (formID) => {
+    if (VALID_ENDPOINTS.includes(formID)) {
+      trackFormAction(formID);
+      setCheckedOptions(checkedFields);
+      setInputResponses(inputQuestions);
+      setSource();
+      const submission = { id: endpoint };
```

Comment:
Is there a reason you are setting two of the values within submission separately? Couldn't you just write it as: `const submission = 
{ id: endpoint, 
formID: formData}`

Reply:
Hmmm the reason I didn't write it like that was because if you just do formID directly, it sets the property to "formID" instead of what formID actually is (e.g. it would be `{id: endpoint, formID: {...}}` rather than `{id: endpoint, feedback: {...}}`, which doesn't match the Flow schema. However, I just found out you can do something like `{id: endpoint,[formID]: {...}}` in JS ES6, so I've updated it to that!